public/login.php

```diff
@@ -87,60 +86,49 @@
     Language::setLanguageCookie($_GET['language']);
 }
 
+/** Initializations */
+
+// AuthSource Simple
+$as = new Simple($casconfig->getValue('authsource'));
+
+// Ticket Store
 $ticketStoreConfig = $casconfig->getOptionalValue('ticketstore', ['class' => 'casserver:FileSystemTicketStore']);
 $ticketStoreClass = Module::resolveClass($ticketStoreConfig['class'], 'Cas\Ticket');
 /** @var $ticketStore TicketStore */
 /** @psalm-suppress InvalidStringClass */
 $ticketStore = new $ticketStoreClass($casconfig);
 
-$ticketFactoryClass = Module::resolveClass('casserver:TicketFactory', 'Cas\Ticket');
+// Ticket Factory
+$ticketFactoryClass = Module::resolveClass('casserver:TicketFactory', 'Cas\Factories');
 /** @var $ticketFactory TicketFactory */
 /** @psalm-suppress InvalidStringClass */
 $ticketFactory = new $ticketFactoryClass($casconfig);
+
+// Processing Chain Factory
+$processingChaingFactoryClass = Module::resolveClass('casserver:ProcessingChainFactory', 'Cas\Factories');
```

Comment:
What is the purpose of getting the class name this way? (I see ticketStore class was done that way, which makes sense because it is a config option,   but I don't understand TicketFactory using it).

Reply:
I think let's do general refactoring (like we chatted about on Google meet) as separate PRs as we upgrade to using controllers/routes/etc